servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -356,7 +356,7 @@ def page_from_offset(offset, page_size):
     return 1 + (offset / page_size)
 
 
-def cromwell_query_params(query, page, page_size):
+def cromwell_query_params(query, page, page_size, auth_headers):
```

Comment:
Since we don't need them for anything else, it would be nice to evaluate `auth_headers is None` in the calling function and pass in just a boolean here, called `is_cromiam` or something